lib/connections/mongoose.js

```diff
@@ -19,6 +19,7 @@ const createConnection = () => {
   logger.silly('Creating Mongo connection', dbpath, serverOptions);
   return mongoose.createConnection(dbpath, {
     promiseLibrary: Promise,
+    useFindAndModify: false,
```

Comment:
I see no evidence this actually did anything and can probably be killed

Reply:
Yeah probably just kill that.